.travis.yml

```diff
@@ -24,15 +24,16 @@ before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
   - sleep 3
-  - "gem install bundler"
-  - "bundle exec govuk-lint-ruby -R app lib"
   - "psql -c 'create database energy_sparks_test;' -U postgres"
   - "cp config/database.yml.travis config/database.yml"
   - "bundle exec rake db:migrate"
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
+  - bundle exec bundle-audit update
 script:
+  - bundle exec govuk-lint-ruby -R app lib
+  - bundle exec bundle-audit check
```

Comment:
I've moved these two down here so the build "fails" in Travis rather than "errors".